### PR TITLE
Add flag for timestamp creation

### PR DESCRIPTION
**Pull request by steveburton4**

### CODE REVIEW

File: src/main/kotlin/RecordProcessor.kt [hunk 1]
```diff
@@ -42,7 +45,9 @@ open class RecordProcessor(private val validator: Validator, private val convert
                 val qualifiedTableName = "$namespace:$tableName".replace("-", "_")
                 logger.debug("Written record to hbase", "record", getDataStringForRecord(record),
                     "formattedKey", String(formattedKey))
-                hbase.put(qualifiedTableName, formattedKey, record.value(), lastModifiedTimestampLong)
+
```
Comment (mark-r-m) [inline src/main/kotlin/RecordProcessor.kt:48]: extra whitespace?

File: src/integration/kotlin/Kafka2HbaseSpec.kt [hunk 2]
```diff
@@ -16,6 +18,7 @@ import java.io.BufferedReader
 import java.text.SimpleDateFormat
 import java.util.*
 
+
```
Comment (mark-r-m) [inline src/integration/kotlin/Kafka2HbaseSpec.kt:21]: extra whitespace?

File: src/main/kotlin/HbaseClient.kt [hunk 3]
```diff
@@ -21,14 +21,14 @@ open class HbaseClient(val connection: Connection, private val columnFamily: Byt
...
-        while (!success && attempts < Config.Hbase.retryMaxAttempts.toInt()) {
+        while (!success && attempts < Config.Hbase.retryMaxAttempts) {
             try {
                 putVersion(table, key, body, version)
                 success = true
             }
             catch (e: Exception) {
-                val delay = if (attempts == 0) Config.Hbase.retryInitialBackoff.toLong()
-                else (Config.Hbase.retryInitialBackoff.toLong() * attempts * Config.Hbase.retryBackoffMultiplier.toFloat()).toLong()
+                val delay = if (attempts == 0) Config.Hbase.retryInitialBackoff
```
Comment (mark-r-m) [inline src/main/kotlin/HbaseClient.kt:30]: extra whitespace?

Comment (DanielChicot) [reply src/main/kotlin/HbaseClient.kt:30]: No excess whitespace here?

File: src/integration/kotlin/Kafka2HbaseSpec.kt (hunk 2 above)
Comment (DanielChicot) [reply src/integration/kotlin/Kafka2HbaseSpec.kt]: Fixed.

**Pull request merged**